docs/docs/pipeline/authn.md

```diff
@@ -482,6 +482,11 @@ header as the subject for this request.
   request against the token endpoint
   - `give_up_after` (string) timeout
   - `max_delay` (string) time to wait between retries
+- `cache` (object, optional) - Enables caching of requested tokens
+  - `enabled` (bool, optional) - Enable the cache, will use exp time of token to
+    determine when to evict from cache. Defaults to false.
+  - `ttl` (string) - Can override the default behaviour of using the token exp
+    time, and specify a set time to live for the token in the cache.
```

Comment:
Missing `max_tokens` config.

Please clarify that TTL can never override a token's TTL!